debian/control

```diff
@@ -405,14 +405,17 @@ Description: debugging symbols for ceph-mon
 Package: ceph-osd
 Architecture: linux-any
 Depends: ceph-base (= ${binary:Version}),
-         lvm2,
-         parted,
+         ceph-volume (= ${binary:Version}),
          sudo,
          ${misc:Depends},
          ${python:Depends},
          ${shlibs:Depends},
-Replaces: ceph (<< 10), ceph-test (<< 12.2.2-14)
-Breaks: ceph (<< 10), ceph-test (<< 12.2.2-14)
+Replaces: ceph (<< 10),
```

Comment:
i am following the case \# 8 outlined in https://wiki.debian.org/PackageTransition#Package_Transition